easybuild/easyconfigs/e/EMAN2/EMAN2-2.11-foss-2015a-Python-2.7.9.eb

```diff
@@ -0,0 +1,44 @@
+easyblock = 'CMakeMake'
+
+name = 'EMAN2'
+version = '2.11'
+
+homepage = 'http://blake.bcm.edu/emanwiki/EMAN2'
+description = """EMAN2 is the successor to EMAN1. It is a broadly based greyscale scientific image processing suite
+ with a primary focus on processing data from transmission electron microscopes. """
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+source_urls = ['http://ncmi.bcm.edu/ncmi/software/counter_222/software_130/']
+sources = ['eman%(version)s.source.tar.gz']
+
+pyver = '2.7.9'
```

Comment:
I need it in a .9 context, so I'll stick to .9 for now, bumping to .10 is trivial/left to the interested reader